roles/edpm_nova/vars/main.yml

```diff
@@ -20,3 +20,6 @@
 # intended to be modified.
 
 # All variables within this role should have a prefix of "edpm_nova"
+
+# this is const path where nova custom configs will land
+edpm_nova_config_sync_src: /var/lib/openstack/configs/custom/nova/
```

Comment:
please use configs/nova, we don't have sosreports coverage for custom/

https://github.com/sosreport/sos/pull/3414/files#diff-5cbe3417b4fe3c187b14e99f4897ccb987ecd6c3bd2b904266b8b932cef22e21R221